docs/projects.md

```diff
@@ -39,3 +40,42 @@ dronedeployApi.Projects.getCurrentlyViewed()
 ```
 
 [**Full Example**](/projects/example-projects.getcurrentlyviewed.md)
+
+### Projects.team
+
+**Overview**
+
+Returns members who have access to this project.
```

Comment:
You should add some disclaimer about using this method on pages that have projectId

Reply:
And maybe add info about potential error that can be thrown